packages/composites/savax-price/src/endpoint/price.ts

```diff
@@ -0,0 +1,60 @@
+import { AdapterContext, AdapterRequest, AdapterResponse } from '@chainlink/types'
+import { Config, FLOATING_POINT_DECIMALS } from '../config'
+import * as TA from '@chainlink/token-allocation-adapter'
+import { ethers } from 'ethers'
+
+export const supportedEndpoints = ['price']
+
+export const execute = async (
+  input: AdapterRequest,
+  context: AdapterContext,
+  config: Config,
+): Promise<AdapterResponse> => {
+  const usdPerAvax = await getAvaxPrice(input, context)
+  const avaxPooledShares = await getPooledAvaxShares(config)
+  const result = usdPerAvax
+    .mul(avaxPooledShares)
+    .div(ethers.BigNumber.from(10).pow(FLOATING_POINT_DECIMALS))
+    .toString()
+  return {
+    jobRunID: input.id,
+    statusCode: 200,
+    result,
+    data: {
+      result,
+    },
+  }
+}
+
+export const getAvaxPrice = async (
+  input: AdapterRequest,
+  context: AdapterContext,
+): Promise<ethers.BigNumber> => {
+  const _config = TA.makeConfig()
+  const _execute = TA.makeExecute(_config)
+  const allocations = [
+    {
+      symbol: 'AVAX',
+      balance: ethers.BigNumber.from(10).pow(FLOATING_POINT_DECIMALS).toString(),
```

Comment:
Why is this 10? Isn't it getting the price of 1 AVAX?

Reply:
yeah it's 1 AVAX.  Isn't the balance to 18dp?  That's what I see in other places.

Reply:
Yeah the calculation here is basically `10^18`, so looks right.